e-commerce/shop-backend/migrations/001_initial_schema.ts

```diff
@@ -31,7 +31,7 @@ export async function up(knex: Knex): Promise<void> {
   });
 
   await knex.schema.createTable('products', table => {
-    table.increments('product_id').primary();
+    table.integer('product_id').primary();

```

Comment:
Let's make these string UUIDs to be on the safe side if we want more products later